salt/haproxy/init.sls

```diff
@@ -69,6 +69,7 @@ haproxy:
       - {{ pillar['api']['ssl_port'] }}
     - proto:      tcp
 
+{% if not salt['grains.get']('tx_update_migration_available', false) %}
```

Comment:
@inercia the node migration (through update orch) keeps failing here, so this exclusion is a bit of a shot in the dark. do you maybe have an idea why it could fail? for some more context, see the ML: "[caasp-devel] cri not reacting during update orchestration"